aten/src/ATen/Declarations.cwrap

```diff
@@ -3865,6 +3865,26 @@
       kwarg_only: True
     - THTensor* self
 ]]
+[[
+  name: _standard_gamma
+  types:
+    - floating_point
+  backends:
+    - CPU
+  return: argument 0
+  variants:
+    - method
+    - function
+  options:
+    - cname: standard_gamma
+      arguments:
+        - arg: THTensor* output
+          output: True
+        - arg: THGenerator* generator
+          default: THPGenerator_TH_CData(THPDefaultGenerator)
```

Comment:
Do we use this pattern anywhere in ATen? It's a bit weird, because now we'll have a cyclic dependency between THP and ATen.

Reply:
Yes, the other random functions that use a generator do this (and have for a long time).  We avoid an actual cyclic dependency because ATen just searches and replaces this string with its way of getting the default generator (at::globalContext()::defaultGenerator() or whatever it's called).  Once we no longer need these declarations for THP we should change this to be ATen based.

Reply:
Oh, those find/replaces was because cwrap was shared with pytorch. They are no longer shared. This should be changed to be something sensible rather than a weird string replace.

Reply:
I filed https://github.com/pytorch/pytorch/issues/4202 for this.